chat/client-api.md

```diff
@@ -0,0 +1,61 @@
+# Client API
+
+```typescript
+abstract class Client {
+  // ---------- Methods ----------------------------------------------- //
+
+  // initializes the client with persisted storage and a network connection
+  public abstract init(): Promise<void>;
+
+  // register a blockchain account with a public key / returns the public key
+  public abstract register(params: {
+    account: string;
+  }): Promise<string>;
+
+  // queries the default keyserver with a blockchain account / returns the public key
+  public abstract resolve(params: {
+    account: string;
+  }): Promise<string>;
+
+  // sends a chat invite with opening message / returns an invite id
+  public abstract invite(params: {
+    publicKey: string;
+    message: string;
+  }): Promise<number>;
```

Comment:
Now our `Invite` data model contains an `account` but api's invite method does not.
should we add it here or provide a single account on init?